### PR TITLE
Add referrer to global ratings survey

### PR DESCRIPTION
**Pull request by dsmr**
Adds a method to the Global Ratings Survey class to allow us to append the user's window.location.href to Survey Monkey links as a custom variable

### CODE REVIEW

**Comment by benjclark** [comment]
Diana and I paired on this 🙂

File: toolkits/global/packages/global-ratings-survey/view/ratingsSurvey.hbs [hunk 1]
```diff
@@ -15,7 +15,7 @@
 				</div>
 				{{#if questionUrl}}
 					{{#if questionText}}
-						<a href="{{questionUrl}}?id={{id}}" target="_blank">{{questionText}} (opens in a new tab)</a>
+						<a href="{{questionUrl}}" data-ratings-survey="survey-link" target="_blank">{{questionText}} (opens in a new tab)</a>
```
Comment (jpw) [inline toolkits/global/packages/global-ratings-survey/view/ratingsSurvey.hbs:18]: Links to external sites must have `rel="noopener"` [added for security reasons](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener) even though [browser support for implied `noopener` behaviour if opening links in new tabs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility) is pretty good.

**Review by jpw — CHANGES_REQUESTED** [review]
A few things to be looked-at please, thanks 🙂

**Review by jpw — APPROVED** [review]

**Comment by jpw** [comment]
Good stuff, thank you 🚀

**Pull request merged**